pages/index.vue

```diff
@@ -1,9 +1,19 @@
 <template>
   <div>
+    <div class="small-banner">
+      <span class="small-banner__title" sx="{styles.titleStyle}">
+        {{ smallBanner.title }} &nbsp;
+      </span>
+      <div class="small-banner__container">
+        <span>{{ smallBanner.subTitle }} </span>
+        <a href="{{" smallBanner.callToAction.url }}>{{ smallBanner.callToAction.title }}</a>
```

Comment:
Using getters for each property may be a good option to prevent code break if properties not provided for url, title etc

Reply:
small banner properties are coming from getters only